src/currencies.ts

```diff
@@ -0,0 +1,14 @@
+import { ICurrency } from './interfaces'
+
+export const CURRENCIES: ICurrency[] = [
+  {
+    name: 'United States Dollar',
+    symbol: '$',
+    code: 'USD',
+  },
+  {
+    name: 'Peso',
+    symbol: '$',
+    code: 'ARS',
+  },
```

Comment:
I wouldn't add this now, take into account that all of the operations are un USD, so adding Pesos without the necessary refactor will be counterproductive